ceph/rbd/snapshot.py

```diff
@@ -22,7 +22,7 @@ def create(self, args):
         pool_name = args["pool_name"]
         image_name = args["image_name"]
         cmd = f"rbd snap create {pool_name}/{image_name}@{snap_name}"
-        if self.exec_cmd(cmd=cmd):
+        if not self.exec_cmd(cmd=cmd):
```

Comment:
Please revisit this.

Reply:
Sorry, had missed it
Removed it now, It had come when we had created a blueprint in initial commit